apps/cms/tsconfig.app.json

```diff
@@ -6,5 +6,16 @@
   },
   "files": ["src/main.ts"],
   "include": ["src/**/*.d.ts"],
-  "exclude": ["jest.config.ts", "src/**/*.test.ts", "src/**/*.spec.ts"]
+  "exclude": [
+    "vitest.config.ts",
+    "src/**/*.test.ts",
+    "src/**/*.spec.ts",
+    "src/**/*.test.tsx",
+    "src/**/*.spec.tsx",
+    "src/**/*.test.js",
+    "src/**/*.spec.js",
+    "src/**/*.test.jsx",
+    "src/**/*.spec.jsx",
+    "src/test-setup.ts"
```

Comment:
The `test-setup.ts` file doesn't exist anymore, so doesn't need to be referenced here.

Reply:
Ah, good point. The Nx plugin thought that `test-setup` created for Jest integration for Angular was created for Vitest so it created a bunch of incorrect logic around it.